documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -6,48 +6,96 @@
 
-Cruise Control maintains a _cached optimization proposal_ based on the configured default optimization goals.
-Generated from the workload model, the cached optimization proposal is updated every 15 minutes to reflect the current state of the Kafka cluster.
-If you generate an optimization proposal using the default optimization goals, Cruise Control returns the most recent cached proposal.
+An optimization proposal summary shows the proposed scope of changes.
 
-To change the cached optimization proposal refresh interval, edit the `proposal.expiration.ms` setting in the Cruise Control deployment configuration.
-Consider a shorter interval for fast changing clusters, although this increases the load on the Cruise Control server.
+You can return a summary from the command line using `kubectl describe kafkarebalance _<kafka_rebalance_resource_name>_ -n _<project_name>_`.
 
-[[contents-optimization-proposals]]
-[discrete]
-== Contents of optimization proposals
+Use the summary to decide whether to approve or reject an optimization proposal.
+
+Approving an optimization proposal:: You approve the optimization proposal by setting the `strimzi.io/rebalance` annotation of the `KafkaRebalance` resource to `approve`.
+Cruise Control applies the proposal to the Kafka cluster and starts a cluster rebalance operation.
+Rejecting an optimization proposal:: If you choose not to approve an optimization proposal, you can change the optimization goals and generate another proposal.
 
-An optimization proposal consists of two main sections:
+Use optimization proposals to assess the movements required for a rebalance.
+For example, a summary describes inter-broker and intra-broker movements.
+Inter-broker rebalancing moves data between separate brokers.
+Intra-broker rebalancing moves data between disks on the same broker.
+Such information can be useful even if you don't go ahead and approve the proposal.
 
-* The _summary_ is stored in the `status` of the `KafkaRebalance` resource.
-* The _broker load_ is stored in a ConfigMap that contains data as a JSON string.
+You might reject an optimization proposal, or delay its approval, because of the additional load on a Kafka cluster when rebalancing.
 
-The summary provides an overview of the proposed cluster rebalance and indicates the scale of the changes involved.
-The broker load shows before and after values for the proposed rebalance, so you can see the impact on each of the brokers in the cluster.
+In the following example, the proposal suggests the rebalancing of data between separate brokers.
+The rebalance involves the movement of 55 partition replicas, totaling 12MB of data, across the brokers.
+Though the inter-broker movement of partition replicas has a high impact on performance, the total amount of data is not large.
+If the total data was much larger, you could reject the proposal, or time when to approve the rebalance to limit the impact on the performance of the Kafka cluster.
```

Comment:
Again we should mention that the rebalance performance tuning options can help reduce the impact of this data movement. For example if you can stand a longer rebalancing period you can reduce the size of each "batch" of reassignments so that less movements are happening at one time (reducing the load on the cluster).

Reply:
Added

> Rebalance performance tuning options can help reduce the impact of data movement. 
> If you can extend the rebalance period, you can divide the rebalance into smaller batches. 
> Fewer data movements at a single time reduces the load on the cluster.